staging/src/k8s.io/client-go/transport/spdy/spdy.go

```diff
@@ -46,8 +54,9 @@ func RoundTripperFor(config *restclient.Config) (http.RoundTripper, Upgrader, er
 	upgradeRoundTripper := spdy.NewRoundTripperWithConfig(spdy.RoundTripperConfig{
 		TLS:        tlsConfig,
 		Proxier:    proxy,
-		PingPeriod: time.Second * 5,
+		PingPeriod: pingPeriod,
 	})
+	klog.V(9).Infof("PingPeriod is set to %v", pingPeriod)
```

Comment:
maybe debug out and explain this is for SPDY / or keep alive pings.. to make it easier to figure out